CarInspection/src/WebCarInspection/Views/Driver/CreateDriver.cshtml

```diff
@@ -0,0 +1,79 @@
+﻿@model WebCarInspection.Models.DriverViewModel
+
+@{
+    ViewData["Title"] = "Create driver";
+}
+
+<h5 style="margin: 2vw;">Create driver</h5>
+    <div style="width: 30vw; margin: 3vw;">
+        <div class="form-group">
+            @using (Html.BeginForm("CreateDriver", "Driver"))
```

Comment:
Refactor all HTML helpers into Tag helpers.
You can also use: https://github.com/DamianEdwards/TagHelperPack

Reply:
It related to all views.